seurat-find-transfer-anchor.R

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env Rscript
+
+suppressPackageStartupMessages(require(optparse))
+suppressPackageStartupMessages(require(workflowscriptscommon))
+
+#set parsing options
+
+option_list = list(
+ 
+  
+   make_option(
+    c("--query-file"),
+    action = "store",
+    default = NA,
+    type = 'character',
+    help = "place to find query ",
```

Comment:
I would go for something like query path or path to the query file
```suggestion
    help = "Query path",
```